src/editor/scene/scene-factory.ts

```diff
@@ -94,6 +94,13 @@ export default class SceneFactory {
 
         return SceneManager.EnvironmentHelper;
     }
+	
+	public static CreateCamera (editor: Editor): Camera {
+        const camera =  new BABYLON.FreeCamera('New Camera', new BABYLON.Vector3(0, 0, 0), editor.core.scene, true);
+        this.AddToGraph(editor, camera);
```

Comment:
Please also add tags to camera like in other functions:
Tags.AddTagsTo(camera, 'added');

Reply:
I don't understand what you mean by this